test/simulator/timestamp/call_log_manager/call_log_manager.cpp

```diff
@@ -167,24 +167,42 @@ call_log_manager::call_log_query_timestamp(const json &call_log_entry)
     /* Convert the config char * to a string object. */
     std::string config = call_log_entry["input"]["config"].get<std::string>();
 
-    /*
-     * A generated call log without a configuration string in the set timestamp entry will have the
-     * string "(null)", default to all_durable.
-     */
-    if (config == "(null)")
-        config = "get=all_durable";
-
+    std::string class_name = call_log_entry["class_name"].get<std::string>();
```

Comment:
You can declare this `string` as `const`:
```suggestion
    const std::string class_name = call_log_entry["class_name"].get<std::string>();
```

Reply:
Fixed.